style.css

```diff
@@ -1,136 +1,690 @@
+  padding: 10px 12px;
+  align-items: center;
+  background-color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.item {
+  margin: 12px;
+}
+
+.work-item-1 {
+  margin-left: 24px;
+  width: 49px;
+  height: 40px;
+  flex: none;
+  order: 0;
+}
+
+.work-item-2 {
+  width: 56px;
+  height: 40px;
+  flex: none;
+  order: 1;
+}
+
+.work-item-3 {
+  width: 95px;
+  height: 40px;
+  flex: none;
+  order: 2;
+}
+
+.work-item-4 {
+  width: 61px;
+  height: 40px;
+  flex: none;
+  order: 3;
+}
+
+.work-link {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 124px;
+  height: 48px;
+  padding: 12px;
+  gap: 10px;
+  background-color: #ff6b00;
+  margin-left: 24px;
+  margin-right: 227px;
+  margin-bottom: 24px;
+}
+
+.work-link,
+.see-project {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+  padding: 12px 12px;
+}
+
+.project {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  left: 0;
+  background-image: url(./Pictures/ImgPlaceholder1.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+}
+
+.project-heading h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 12px;
+  margin-top: 84px;
+  text-align: justify;
+}
+
+.project-heading p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  text-align: justify;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 12px;
+}
+
+.project-lists {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  align-content: space-between;
+}
+
+.project-lists ul {
+  list-style: none;
+  display: flex;
+  flex-grow: 0;
+}
+
+.project-lists li {
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  background-color: rgba(255, 255, 255, 0.24);
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  padding: 10px 12px;
+  color: #fff;
+  margin-bottom: 22px;
+}
+
+.project-item-1 {
+  display: flex;
+  margin-left: 16px;
+  margin-right: 12px;
+  width: 56px;
+  height: 40px;
+  order: 0;
+}
+
+.project-item-2 {
+  display: flex;
+  margin-right: 12px;
+  width: 95px;
+  height: 40px;
+  order: 1;
+}
+
+.project-item-3 {
+  display: flex;
+  margin-right: 75px;
+  width: 61px;
+  height: 40px;
+  order: 2;
+}
+
+.project-item {
+  margin: 12px;
+}
+
+.project-list {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  background-color: #ff6b00;
+  width: 100%;
+  height: 48px;
+  left: 0;
+  top: 0;
+}
```

Comment:
Amazing job adjusting the margin and the button width 😄! Please remove the underline on the button text to make it fit the one in Figma 😉.
![image](https://user-images.githubusercontent.com/42799579/198765765-225b8cfd-8127-41a5-a3b2-8da57485aba3.png)